### ISSUE TITLE
Sklearn missing from dvcorg/cml-py3:latest

### ISSUE THREAD

**Issue by ivyleavedtoadflax** [iterative/example_cml#18]
I'm following the tutorial here, and ran into the following issue:

```
Run # Your ML workflow goes here
Traceback (most recent call last):
  File "train.py", line 1, in <module>
    from sklearn.ensemble import RandomForestClassifier
ModuleNotFoundError: No module named 'sklearn'
##[error]Process completed with exit code 1.
```

You can see my attempts to get this to work in this PR: https://github.com/muanalytics/example_cml/pull/1

Following the tutorial without making any changes brought me to this error: https://github.com/muanalytics/example_cml/runs/875446393

I tweaked the cml.yaml and you'll see that I got it working in the end. I was expecting this to work out of the box though?

**Comment by ivyleavedtoadflax** [comment]
Ah sorry classically copied cml.yaml from the wrong place:

it should have had the `pip install -r ...`

**Comment by ivyleavedtoadflax** [comment]
Probably is worth correcting the `results.txt` error in the first `cml.yaml` code block here: https://github.com/iterative/cml#getting-started

**Comment by ivyleavedtoadflax** [comment]
This is the first one: 

![image](https://user-images.githubusercontent.com/4583655/87611626-841cc380-c6d6-11ea-98d8-d004e500720e.png)

This is the second:

![image](https://user-images.githubusercontent.com/4583655/87611658-9565d000-c6d6-11ea-9695-6b67e9e4fe6b.png)

Also, love your work! DVC :heart:

**Comment by andronovhopf** [comment]
Hi @ivyleavedtoadflax, thanks for trying it out! If I follow correctly, the first code block is from the Usage section of the README, and the second is from Getting Started. 

The Usage section is intended to demonstrate the simplest possible use case, with some explanation of the big picture. It's not part of the tutorial. Do you think there's anything we could do in the README to make this more clear?

**Comment by ivyleavedtoadflax** [comment]
Hi @andronovhopf thanks for baking great stuff🙏! I think I'd be inclined to at least replicate the `pip install... ` in all places otherwise other users could make the same mistake as me and assume that the docker container already contains all the dependencies required for the tutorial. That said, I should probably have read the whole thing first 🙊

**Comment by andronovhopf** [comment]
You're right, leaving it out could confuse people about what's in the Docker container.  I'm updating the code examples in the README so `pip install...` is always present.

Glad you pointed it out! :)

**Comment by andronovhopf** [comment]
Addressed and closing the issue ([see commit here](https://github.com/iterative/cml/commit/4d87e7a6b477b65ac86ae98fd788eabf0761d192). Please reopen or make a new issue if you have any other trouble!

**Comment by ivyleavedtoadflax** [comment]
Thanks @andronovhopf
